Testing/SystemTests/tests/framework/SANSILLAutoProcessTest.py

```diff
@@ -172,6 +172,59 @@ def runTest(self):
         )
 
 
+class D11_AutoProcess_Multiple_Transmissions_Test(systemtesting.MantidSystemTest):
+    """
+    Tests auto process for D11 with 1 sample at 3 different distances,
+    and with multiple transmissions per process.
+    """
+
+    def __init__(self):
+        super(D11_AutoProcess_Multiple_Transmissions_Test, self).__init__()
+        self.setUp()
+
+    def setUp(self):
+        config['default.facility'] = 'ILL'
+        config['default.instrument'] = 'D11'
+        config['logging.loggers.root.level'] = 'Warning'
+        config.appendDataSearchSubDir('ILL/D11/')
+
+    def cleanup(self):
+        mtd.clear()
+
+    def validate(self):
+        self.tolerance = 1e-3
+        self.tolerance_is_rel_err = True
+        return ['iq_mult_wavelengths', 'D11_AutoProcess_Multiple_Tr_Reference.nxs']
+
+    def runTest(self):
+        beams = '950,1020,947'
+        containers = '958,1023,973'
+        container_tr = '988,1023,988'
+        beam_tr = '947,1020,947'
+        samples = '960,1024,975'
+        sample_tr = '990,1024,990'
```

Comment:
Just to confirm, is it really the middle distance that is at different wavelength? I though it's the first or the last?

Reply:
The first and the last have the same wavelength of 4.6, and the middles is at 9.99, according to the meta-data.